### PR TITLE
Set clearer expectations

### PR DESCRIPTION
**Pull request by nfelger**
I think both we and our applicants would benefit from setting clearer expectations on the coding challenge. We're less interested in any gotchas applicants might find and more in receiving a work sample that is close to what they would produce if they worked for us.

Therefore, two review questions:
1. What do you think of this generally, and
2. would you change anything about the list I drafted?

### CODE REVIEW

**Comment by nfelger** [comment]
@zechmeister @frederike-ramin @JulianRoesner @joschka @SannyNguyenHung Please feel free to comment / review as well, if you're interested in this topic!

File: README.md [hunk 1]
```diff
@@ -13,7 +13,19 @@ Git during building your application please make sure to include the Git history
...
 
-You should turn in a complete and runnable solution, but do not gold-plate it: we don't expect you to spend more than a few hours on this exercise. We want to see that you know how to use APIs efficiently, how you structure and how you are testing your application. It should be easy to tell from the dashboard which ministries have provided the most data.
+We want to use this challenge to understand how you would approach such a problem in a work setting. Here are some specifics of what we are looking for:
+
+Among other things, we pay attention to:
+- how you structure your application and your source code,
+- how you name variables/functions/classes/packages,
+- how you test your application and what kinds of tests you're writing,
+- how you document your application, including instructions to run the project,
+- how you write your git commit messages,
```
Comment (benjaminaaron) [inline README.md]: Further up in the text it sounds like using git is optional though. Maybe make it mandatory (I'd vote for that) or also add "if you use git" here?

Comment (carhartl) [reply README.md]: Let's generalize this some more - it's not just about the commit message, but whether a developer commits atomic changesets or various unrelated changes in a single huge commit. There's some correlation here: atomic commits will allow you to write much better commit messages, as you need to describe only one thing at a time..

Comment (carhartl) [reply README.md]: So, something like: "How your commit log looks like" ... 🤔

Comment (nfelger) [reply README.md]: Gotcha. I've made VCS mandatory (doesn't have to be git, I think?) and have generalized the expectation to "what your commits look like". Does that sit right with you @benjaminaaron @carhartl ?

**Review by carhartl — APPROVED** [review]
👍

**Comment by chrkaatz** [comment]
I rewrote the parts or the submission - I am not a huge fan of the zip files :-)

I like the change to be more precise :-)

**Review by chrkaatz — APPROVED** [review]
Looks good!

**Comment by nfelger** [comment]
> I rewrote the parts or the submission - I am not a huge fan of the zip files :-)

Good shout, but isn't it a bit confusing now between local git and hosted (github) repos? I wouldn't be sure how I'm supposed to share my private repo with… Tried making it a bit clearer. @chrkaatz care to take another look?

**Comment by chrkaatz** [comment]
@nfelger Can we look at this later together?

**Comment by chrkaatz** [comment]
Now it looks great!

**Pull request merged**